control-plane/plugin/src/resources/node.rs

```diff
@@ -239,6 +336,126 @@ impl GetHeaderRow for NodeDisplay {
                 header.extend(vec!["CORDON LABELS"]);
                 header
             }
+            NodeDisplayFormat::Drain => {
+                header.extend(vec!["DRAIN STATE"]);
+                header.extend(vec!["DRAIN LABELS"]);
+                header
+            }
+        }
+    }
+}
+
+#[derive(Debug, Clone, clap::Args)]
+pub struct DrainNodeArgs {
+    /// Id of the node
+    node_id: NodeId,
+    /// Label of the drain
+    label: String,
+    #[clap(long)]
+    /// Timeout for the drain operation
+    drain_timeout: Option<humantime::Duration>,
+}
+
+impl DrainNodeArgs {
+    /// Return the node ID.
+    pub fn node_id(&self) -> NodeId {
+        self.node_id.clone()
+    }
+    /// Return the drain label.
+    pub fn label(&self) -> String {
+        self.label.clone()
+    }
+    /// Return whether or not we should show the drain labels.
+    pub fn drain_timeout(&self) -> Option<humantime::Duration> {
```

Comment:
The comment doesn't seem to go with the actual operation

Reply:
fixed